starter/script.js

```diff
@@ -1 +1,122 @@
 // put code here!
+var endgame = false
+
+while (endgame === false) {
```

Comment:
This isn't wrong, but when we know that our variable will only be `true` or `false` , instead of `variable_name === false`, we can more simply test for `false` with `!variable_name`